Library/Homebrew/utils.rb

```diff
@@ -124,20 +114,20 @@ def odie(error)
 def pretty_installed(f)
   if !$stdout.tty?
     "#{f}"
-  elsif ENV["HOMEBREW_NO_EMOJI"]
+  elsif !Emoji.enabled?
     "#{Tty.highlight}#{Tty.green}#{f} (installed)#{Tty.reset}"
   else
-    "#{Tty.highlight}#{f} #{Tty.green}#{Tty.tick}#{Tty.reset}"
+    "#{Tty.highlight}#{f} #{Tty.green}#{Emoji.tick}#{Tty.reset}"
   end
 end
 
 def pretty_uninstalled(f)
   if !$stdout.tty?
     "#{f}"
-  elsif ENV["HOMEBREW_NO_EMOJI"]
+  elsif !Emoji.enabled?
```

Comment:
Would you mind swapping the `else` and `elsif` bodies to avoid the negation in this condition?